readme.txt

```diff
@@ -8,6 +8,7 @@
 * Tweak - Add the `tribe_template_done` filter to be able to disable a template before rendering. [TEC-3385]
 * Fix - Blocks editor CSS compatibility with WordPress 5.4 with new module classes: `.block-editor-inner-blocks`
 * Fix - Add style override for <ul> in Divi due to theme use of IDs [TEC-3235]
+* Fix - Change text domain loading to occur on 'init' hook instead of 'plugins_loaded'. Added new `tribe_load_text_domain` action hook for our other plugins to use for their own text domain loading on 'init' as well. [TCMN-58]
```

Comment:
```suggestion
* Fix - Change text domain loading to occur on 'init' hook instead of 'plugins_loaded'. Added new `tribe_load_text_domains` action hook for our other plugins to use for their own text domain loading on 'init' as well. [TCMN-58]
```